slides/package.json

```diff
@@ -8,6 +8,7 @@
     "clean": "rm -rf dist",
     "build": "npm run clean && webpack",
     "start": "webpack-dev-server",
+    "test": "cd testing && tsc index.ts --outdir dist -t es6 -m commonjs && cd dist && node index.js",
```

Comment:
Do you need the `cd` commands? Can you not do a relative path reference like:
`"test": "tsc testing/index.ts --outdir testing/dist -t es6 -m commonjs && node dist/index.js",`